site/content/3.10/arangograph/oasisctl/accept/_index.md

```diff
@@ -1,37 +1,30 @@
 ---
+layout: default
```

Comment:
Is this needed? We don't set a layout for most of the content pages - and for the few where we do, it might be by mistake. In Jekyll, this was necessary and we did it for all pages. But in Hugo?

Reply:
@dandimeo can we safely remove `layout`? or is this needed?

Reply:
addressed via https://github.com/arangodb/docs-hugo/pull/319